### DISCUSSION TITLE
Welcome to FastSpeech2 Discussions!

### DISCUSSION THREAD

**Discussion by rishikksh20** [rishikksh20/FastSpeech2#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by luudinh93** [comment]
i am training your model but getting error at step hs=hs+self.pitch_embeded because their dimensionality is different. Please help me fix this error
![image](https://user-images.githubusercontent.com/4282279/155519759-4336a3e5-772a-4085-beac-86d5e1c698c3.png)

**Reply by rishikksh20** [reply]
length of hidden state (1261) and length of the pitch(1119) is different, both should be same either 1261 or 1119. Please check your mel files

**Reply by luudinh93** [reply]
I have fixed the above error but there is another error
![image](https://user-images.githubusercontent.com/4282279/155983078-a7bfaa09-105b-4a30-b836-385d13328ec1.png)

**Reply by rishikksh20** [reply]
check your duration for respective phonemes I think some duration in your pre-processed file is negative

**Reply by luudinh93** [reply]
I guess I created the filelist file the wrong way. Please give me detailed instructions for this step
![image](https://user-images.githubusercontent.com/4282279/156081336-1eaaa477-009f-4ace-a870-f47beb82887d.png)
**Example:**
`tgs, tss = textgrid.TextGrid.fromFile(path_to_textgrid)`

`tgs = IntervalTier(words, [Interval(0.0, 0.46, None), Interval(0.46, 0.64, chú), Interval(0.64, 1.09, thích), Interval(1.09, 1.11456, None)])`

`tts = IntervalTier(phones, [Interval(0.0, 0.46, None), Interval(0.46, 0.55, ch), Interval(0.55, 0.64, u1_T3), Interval(0.64, 0.73, th), Interval(0.73, 0.81, i_T3), Interval(0.81, 1.09, ch), Interval(1.09, 1.11456, None)])`

**File that I created: 
metadata text:** audio_001.wav|chú thích. 
`<unk> chú thích <unk>| -2 0 46 55 64 73 81 109|2 46 9 9 9 8 28|pau ch u1_T3 th i_T3 ch pau|audio_001.wav`
